peerconnection.go

```diff
@@ -1099,6 +1099,11 @@ func (pc *PeerConnection) openSRTP() {
 			return
 		}
 
+		if pc.currentLocalDescription == nil {
```

Comment:
Should this be after we take the lock?

Reply:
I wasn't sure abiut that, I thought it could probably be either. My reasoning behind putting it there is that we can do the check as soon as we return from the previous function call, so no need to wait for the lock.